Sources/Core/Sources/ECS/Singles/InputState.swift

```diff
@@ -94,28 +94,20 @@ public final class InputState: SingleComponent {
   /// Ticks the input state by flushing ``newlyPressed`` into ``keys`` and ``inputs``, and clearing
   /// ``newlyReleased``. Also emits events to the given ``EventBus``.
   func tick(_ isInputSuppressed: [Bool], _ eventBus: EventBus) {
-    /// increment the tick count
+    // increment the tick count
     tickCount += 1
-    /// increment the time since the forwards key was pressed if it is currently pressed
+    // increment the time since the forwards key was pressed if it is currently pressed
     assert(isInputSuppressed.count == newlyPressed.count, "`isInputSuppressed` should be the same length as `newlyPressed`")
     for (var event, suppressInput) in zip(newlyPressed, isInputSuppressed) {
       if suppressInput {
         event.input = nil
       }
-      ///test for forwards key
+      // test for forwards key
       if event.input == .moveForward {
-        /// if the forwards key has been pressed, released, and pressed again within 6 ticks, sprint
-        if forwardsDownTime < forwardsUpTime && (forwardsDownTime + 6) > tickCount {
+        // if the forwards key has been released within 6 ticks, sprint
+        if (forwardsUpTime + 6) > tickCount {
```

Comment:
Use `>=` instead of `>` so that it triggers up to 6 ticks after releasing forwards (instead of 5).